clj-refactor.el

```diff
@@ -3254,6 +3254,17 @@ if REMOVE-PACKAGE_VERSION is t get rid of the (package: 20150828.1048) suffix."
         (replace-regexp-in-string " (.*)" "" version)
       version)))
 
+(defcustom get-cljr-version (lambda ()
```

Comment:
Why don't we make this a string that just defaults to the result of `cljr--version`? Also a `defcustom`'s name should start with `cljr`, so I'd name this `cljr-middleware-version` or something along those lines.

Reply:
Or rather `cljr-injected-middleware-version`. Not sure how I named this in CIDER, but it'd be nice for us to be consistent across the packages.